src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -664,26 +665,37 @@ public String toVariableNameString(final Inventory i) {
 								"The first requires that the player is online and also accepts only part of the name, " +
 								"while the latter doesn't require that the player is online, but the player's name has to be entered exactly.")
 				.usage("")
```

Comment:
Use the usage to explain how this class parses. Explain how it will select the player whom matches the start of their name and that multiple players cannot return from the parse.

Mirror this for OfflinePlayer